parsely/parselyandroid/ParselyTracker.java

```diff
@@ -217,9 +198,11 @@ private void persistQueue(){
 
             this.persistObject(storedQueue);
         }
+        this.persistObject(storedQueue);
```

Comment:
The logic in this function has gotten a little messy. The second null check appears to be redundant, as is the first call to `persistObject`.